addons/rook/1.0.4/build-images/ceph/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # https://github.com/ceph/ceph-container
 
-FROM ceph/ceph:v14.2.0
+FROM ceph/ceph:v14.2.21
```

Comment:
It probably makes sense to add this as a new addon version like `1.0.4-14.2.21`. Image tag would also need to change.